### ISSUE TITLE
JSX boolean attribute "readOnly" should not use string values

### ISSUE THREAD

**Issue by locomotif** [swagger-api/swagger-ui#4917]
### Q&A (please complete the following information)
 - OS: macOS
 - Browser: Chrome
 - Version: 69.0.3497.100 (Official Build) (64-bit)
 - Method of installation: yarn
 - Swagger-UI version: 3.19.0
 - Swagger/OpenAPI version: 3.19.0

### To reproduce...
This is not necessarily a bug, but maybe a react best practice. When executing a sample rest endpoint, I am noticing a warning on the console.  Please see screenshot below.  The issue originates from `swagger-ui/src/core/components/curl.jsx`

___Please note the `readOnly` attribute___
```js
<textarea onFocus={this.handleFocus} readOnly="true" className="curl" style={{ whiteSpace: "normal" }} value={curl}></textarea>
// ...should be
<textarea onFocus={this.handleFocus} readOnly className="curl" style={{ whiteSpace: "normal" }} value={curl}></textarea>
```

### Expected behavior

I expect no warnings

### Screenshots

![image](https://user-images.githubusercontent.com/334640/46373489-00016500-c643-11e8-8546-e64ea70e6bdf.png)

**Comment by hkosova** [comment]
PR: #5497

**Comment by shockey** [comment]
fixed in #5497!
